documentation/api/io.strimzi.api.kafka.model.KafkaAuthorizationSimple.adoc

```diff
@@ -0,0 +1,27 @@
+Simple authorization in Strimzi uses the `AclAuthorizer` plugin, the default Access Control Lists (ACLs) authorization plugin provided with Apache Kafka.
+ACLs allow you to define which users have access to which resources at a granular level.
+
+To use simple authorization, set the `type` property in the `authorization` section to the value `simple`,
+and configure super users.
```

Comment:
Should we clarify that it's in the Kafka custom resource?

Reply:
:+1: 

> Configure the `Kafka` custom resource to use simple authorization.
> Set the `type` property in the `authorization` section to the value `simple`,
> and configure a list of super users.